op-acceptance-tests/tests/isthmus/operator_fee/operator_fee_test.go

```diff
@@ -0,0 +1,304 @@
+package isthmus
+
+import (
+	"log/slog"
+	"math/big"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/devnet-sdk/system"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/systest"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/testing/testlib/validators"
+	"github.com/ethereum-optimism/optimism/devnet-sdk/types"
+	"github.com/ethereum-optimism/optimism/op-e2e/bindings"
+	"github.com/ethereum-optimism/optimism/op-node/rollup"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum-optimism/optimism/op-service/testlog"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/params"
+	"github.com/stretchr/testify/require"
+)
+
+// TestFees verifies that L1/L2 fees are handled properly in different fork configurations
+func TestOperatorFee(t *testing.T) {
+	logger := testlog.Logger(t, slog.LevelDebug)
+	// Define which L2 chain we'll test
+	chainIdx := uint64(0)
+
+	logger.Info("Starting operator fee test", "chain", chainIdx)
+
+	// Get validators and getters for accessing the system and wallets
+	l1WalletGetter, l1WalletValidator := validators.AcquireL1WalletWithFunds(types.NewBalance(big.NewInt(params.Ether)))
+	l2WalletGetter, l2WalletValidator := validators.AcquireL2WalletWithFunds(chainIdx, types.NewBalance(big.NewInt(params.Ether)))
+
+	logger.Info("Acquired test wallets with funds")
+
+	// Run isthmus test
+	_, forkValidator := validators.AcquireL2WithFork(chainIdx, rollup.Isthmus)
+	nodesValidator := validators.HasSufficientL2Nodes(chainIdx, 2)
+	logger.Info("Running system test", "fork", "Isthmus", "nodes", 2)
+	systest.SystemTest(t,
+		func(t systest.T, sys system.System) {
+			logger.Info("Starting operator fee test scenario", "chain", chainIdx)
+			// Get the low-level system and wallet
+			l1Wallet := l1WalletGetter(t.Context())
+			l2Wallet := l2WalletGetter(t.Context())
+			logger.Info("Acquired wallets",
+				"l1_wallet", l1Wallet.Address().Hex(),
+				"l2_wallet", l2Wallet.Address().Hex())
+
+			// get l2WalletBalance
+			l2GethSeqClient, err := sys.L2s()[chainIdx].Nodes()[0].GethClient()
+			require.NoError(t, err)
+			l2WalletBalance, err := l2GethSeqClient.BalanceAt(t.Context(), l2Wallet.Address(), nil)
+			require.NoError(t, err)
+			logger.Info("L2 wallet balance", "balance", l2WalletBalance)
+
+			// Define test cases with different operator fee parameters
+			testCases := GenerateAllTestParamsCases(1, l2WalletBalance)
```

Comment:
Should the first param be `1`?
Related to that, should we make it a named variable so its purpose is more self-descriptive?

Reply:
Fixed in 4ddba19a657fd0d4c7d7e54bbf25c7330081ec2a